common/datasets/switchboard/corpus_train.py

```diff
@@ -0,0 +1,179 @@
+"""
+Functions related to speaker and training corpus creation
+"""
+__all__ = [
+    "get_train_bliss_corpus_ldc",
+    "get_train_bliss_corpus_i6_legacy",
+    "get_train_corpus_object_ldc",
+    "get_train_corpus_object_i6_legacy",
+]
+
+import os
+from sisyphus import tk
+
+from i6_core.tools.download import DownloadJob
+from i6_core.datasets.switchboard import (
+    DownloadSwitchboardTranscriptionAndDictJob,
+    CreateSwitchboardBlissCorpusJob,
+    SwitchboardSphereToWaveJob,
+    DownloadSwitchboardSpeakersStatsJob,
+    CreateSwitchboardSpeakersListJob,
+    CreateLDCSwitchboardSpeakerListJob,
+)
+
+from i6_experiments.common.datasets.util import CorpusObject
+
+from .constants import SUBDIR_PREFIX, durations
+from .paths import SWITCHBOARD1_PATH, SWITCHBOARD1_LEGACY_PATH
+
+
+def get_train_bliss_corpus(
+    audio_dir: tk.Path,
+    skip_empty_ldc_file: bool = False,
+    subdir_prefix: str = SUBDIR_PREFIX,
+) -> tk.Path:
+    """
+    Returns Switchboard training bliss corpus
+
+    :param audio_dir: path for audio data
+    :param subdir_prefix: alias name prefix
+    :param skip_empty_ldc_file: exclude sw2167B which is empty in the official LDC download
+    :return: Path to switchboard training corpus
+    """
+    swb_trans_and_dict = DownloadSwitchboardTranscriptionAndDictJob()
+    swb_trans_and_dict.add_alias(
+        os.path.join(subdir_prefix, "download_trans_and_dict_job")
+    )
+
+    speakers_list = get_speakers_list_legacy(subdir_prefix=subdir_prefix)
```

Comment:
Is it correct that the legacy speaker list is always used here? When calling `get_train_bliss_corpus()` (as opposed to `..._i6_legacy()`), users might not expect to get legacy behavior.

Reply:
I will remove this whole function.